README.md

```diff
@@ -147,3 +147,32 @@ Click the `Allow Anyway` button.
 Building and testing is most easily performed with `make build` and `make test` respectively.
 
 The PGP key used for encrypting the test cases is found in `test/testing-key.pgp`. You can import it with `gpg --import test/testing-key.pgp`.
+
+## Transitioning to Terraform 0.13 (0.12.20+) provider required blocks.
+
+With Terraform 0.13 (0.12.20+), providers are available/downloaded via the [terraform registry](https://registry.terraform.io/providers/carlpett/sops/latest) via a required_providers block (Yay!)
```

Comment:
I think we can just call this 0.13 for simplicity. Even though 0.12.20+ formally supports the `source` field, it appears to panic if you set `source`: https://github.com/hashicorp/terraform/blob/v0.12.20/configs/module.go#L173-L178
(And the same behaviour is kept up until 0.12.29 at least)